src/Uno.UI.Tests/Windows_UI_Input/Given_GestureRecognizer.cs

```diff
@@ -193,15 +193,16 @@ public void Manipulation_Begin()
 		{
 			var sut = new GestureRecognizer { GestureSettings = GestureSettingsHelper.Manipulations };
 			var result = new ManipulationRecorder(sut);
-			var step = GestureRecognizer.MinManipulationDeltaTranslateX;
+			var step = GestureRecognizer.Manipulation.StartTouch.TranslateX;
```

Comment:
May there's a null-ref risk here?

Reply:
No it's only static  readonly properties